common/components/TXMetaDataPanel/components/AdvancedGas.tsx

```diff
@@ -74,7 +74,7 @@ class AdvancedGas extends React.Component<Props, State> {
             <div className="input-group-wrapper AdvancedGas-gas-price">
               <label className="input-group">
                 <div className="input-group-header">
-                  {translate('OFFLINE_Step2_Label_3')} (gwei)
+                  {translateRaw('OFFLINE_Step2_Label_3')} (gwei)
```

Comment:
I'd like to see a better key for this one, the `OFFLINE_stepX` keys are hard to parse at a glance.